### ISSUE TITLE
WSDDN CorLoc

### ISSUE THREAD

**Issue by zhaohui-yang** [mansimane/visual_learning_and_recognition#1]
Hi,
    I found this wsddn code has some trival differences with the paper. For example, the spatial regularizer, the learning rate and etc. So I was wondering if you have the trained model, and tested the corloc of it? Is it comparable with the paper?
